From 7e8d842b9161f44e74da9d0d9fd662e9bc3b041b Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Thu, 12 Aug 2021 22:41:31 +0000
Subject: [PATCH] more clang optimizations

Change-Id: I0d0eaf2ad039d86a16f5e383e533b32b562e11b7
---
 Android.bp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Android.bp b/Android.bp
index 83cc015ec..24a7d15b1 100644
--- a/Android.bp
+++ b/Android.bp
@@ -138,6 +138,7 @@ cc_library {
         "-Werror",
         "-Wno-unused-parameter",
         "-Wno-missing-field-initializers",
+        "-O3", "-g0",
     ],
     lto: {
         thin: true,
-- 
2.30.2

