From 8aea9b57755587e9ebab66bb74f8565a5e982f3d Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sat, 27 Feb 2021 10:53:43 -0500
Subject: [PATCH 5/5] totally remove in-built FOD menu

---
 .../settings/fragments/LockScreen.java        | 55 -------------------
 1 file changed, 55 deletions(-)

diff --git a/src/com/crdroid/settings/fragments/LockScreen.java b/src/com/crdroid/settings/fragments/LockScreen.java
index 078500a0..34d6e15a 100644
--- a/src/com/crdroid/settings/fragments/LockScreen.java
+++ b/src/com/crdroid/settings/fragments/LockScreen.java
@@ -34,7 +34,6 @@ import androidx.preference.Preference.OnPreferenceChangeListener;
 import androidx.preference.SwitchPreference;
 
 import com.android.internal.logging.nano.MetricsProto;
-import com.android.internal.util.crdroid.FodUtils;
 
 import com.android.settings.R;
 import com.android.settings.SettingsPreferenceFragment;
@@ -58,24 +57,11 @@ public class LockScreen extends SettingsPreferenceFragment
     private static final String FP_SUCCESS_VIBRATE = "fp_success_vibrate";
     private static final String FP_ERROR_VIBRATE = "fp_error_vibrate";
 
-    private static final String FOD_ICON_PICKER_CATEGORY = "fod_icon_picker_category";
-    private static final String FOD_ICON_PICKER = "fod_icon_picker";
-    private static final String FOD_GESTURE = "fod_gesture";
-    private static final String FOD_RECOGNIZING_ANIMATION = "fod_recognizing_animation";
-    private static final String FOD_ANIM = "fod_anim";
-    private static final String FOD_NIGHT_LIGHT = "fod_night_light";
-    private static final String FOD_COLOR = "fod_color";
-    private static final String FOD_FOOTER = "fod_footer";
-
     private static final String LOCKSCREEN_BLUR = "lockscreen_blur";
 
-    private PreferenceCategory mFODIconPickerCategory;
-
     private Preference mFingerprintVib;
     private Preference mFingerprintVibErr;
     private Preference mLockscreenBlur;
-    private Preference mScreenOffFOD;
-    private Preference mFODnightlight;
 
     @Override
     public void onCreate(Bundle savedInstanceState) {
@@ -85,26 +71,6 @@ public class LockScreen extends SettingsPreferenceFragment
         PreferenceScreen prefSet = getPreferenceScreen();
         Context mContext = getContext();
 
-        mFODIconPickerCategory = findPreference(FOD_ICON_PICKER_CATEGORY);
-        if (!FodUtils.hasFodSupport(getContext())) {
-            prefSet.removePreference(mFODIconPickerCategory);
-        } else {
-            findPreference(FOD_FOOTER).setTitle(R.string.fod_pressed_color_footer);
-
-            mScreenOffFOD = (Preference) findPreference(FOD_GESTURE);
-            final boolean isScreenOffFodSupported = mContext.getResources().getBoolean(
-                    R.bool.config_supportScreenOffFod);
-            if (!isScreenOffFodSupported) {
-                mFODIconPickerCategory.removePreference(mScreenOffFOD);
-            }
-            mFODnightlight = (Preference) findPreference(FOD_NIGHT_LIGHT);
-            final boolean isFodNightLightSupported = mContext.getResources().getBoolean(
-                    com.android.internal.R.bool.disable_fod_night_light);
-            if (!isFodNightLightSupported) {
-                mFODIconPickerCategory.removePreference(mFODnightlight);
-            }
-        }
-
         PreferenceCategory gestCategory = (PreferenceCategory) findPreference(LOCKSCREEN_GESTURES_CATEGORY);
 
         FingerprintManager mFingerprintManager = (FingerprintManager)
@@ -181,27 +147,6 @@ public class LockScreen extends SettingsPreferenceFragment
                         keys.add(FP_ERROR_VIBRATE);
                     }
 
-                    if (!FodUtils.hasFodSupport(context)) {
-                        keys.add(FOD_ICON_PICKER_CATEGORY);
-                        keys.add(FOD_ICON_PICKER);
-                        keys.add(FOD_GESTURE);
-                        keys.add(FOD_RECOGNIZING_ANIMATION);
-                        keys.add(FOD_ANIM);
-                        keys.add(FOD_NIGHT_LIGHT);
-                        keys.add(FOD_COLOR);
-                    } else {
-                        final boolean isScreenOffFodSupported = context.getResources().getBoolean(
-                                R.bool.config_supportScreenOffFod);
-                        if (!isScreenOffFodSupported) {
-                            keys.add(FOD_GESTURE);
-                        }
-                        final boolean isFodNightLightSupported = context.getResources().getBoolean(
-                                com.android.internal.R.bool.disable_fod_night_light);
-                        if (!isFodNightLightSupported) {
-                            keys.add(FOD_NIGHT_LIGHT);
-                        }
-                    }
-
                     if (!DeviceUtils.isBlurSupported()) {
                         keys.add(LOCKSCREEN_BLUR);
                     }
-- 
2.30.2

